notebooks/test_2.ipynb

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print('Hello anthony.')
</pre><p>As this is a person's name it should be capitalized.(you can comment like this on a specific line of a code block)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/moaatt2/reviewnb-test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I will fix this and update it.</p>

Reply:
<p>(reviewee): I have fixed the issue please review.</p>

Reply:
<p>(reviewer): I see the capitalization change.</p><p> </p><p>I do find this odd that a new conversation had to be made and it didn't recognize that the previous was a cell update. Though these things do happen some time.</p>